stories/Components/ScrollContainer/ScrollContainer.mdx

```diff
@@ -0,0 +1,130 @@
+import { Meta, Story, Canvas } from '@storybook/blocks';
+import * as ScrollContainerStories from './ScrollContainer.stories';
+import ScrollContainer from './ScrollContainer.jsx';
+import './scroll-container.scss';
```

Comment:
Maybe you have this temporarily for debug, but we'll need to remove it before we merge as it will break the CSS import chain (won't respect the Sass Variable hierarchy for the other themes)

Reply:
I mean just line 4 "import './scroll-container.scss';"